### ISSUE TITLE
Discrepancy in PDF instructions for Eng1

### ISSUE THREAD

**Issue by illo1620** [elastic/docs#957]
Here's the details on the typo that Kayla found in the lab instructions for Engineer I...

There are two ways to get the lab instructions:
From a trainee's Strigo enviornment (ex: my page was http://ec2-54-93-233-34.eu-central-1.compute.amazonaws.com/instructions/labs.html)
Or from the 'Elasticsearch Engineer I Lab Kit' ZIP download from the trainee's training.elastic.co class page.

For Lab 12, Step 3 - the instructions from the Strigo environment have a typo in the POST command. Strigo has the INCORRECT command:
POST access_logs/_doc/ 
{
[...]
The Engineer I Lab Kit ZIP file has the CORRECT command (doc with NO underscore):
POST access_logs/doc/ 
{
[...]

I've attached a couple screenshots that illustrate the problem you run into when you run the incorrect command.

In short, the Strigo labs.html file should be changed to match the training.elastic.co labs.html.

![correct](https://user-images.githubusercontent.com/47541672/60304379-443a4100-9907-11e9-8c6e-e27b46a6023c.png)
![incorrect](https://user-images.githubusercontent.com/47541672/60304381-443a4100-9907-11e9-9a38-33edbc2400d7.png)
